metaflow/client/core.py

```diff
@@ -1501,6 +1543,27 @@ def _iter_filter(self, x):
         # exclude _parameters step
         return x.id[0] != "_"
 
+    @property
+    def origin_pathspec(self):
+        """
+        Pathspec of the origin run that the current run was cloned from.
+
+        Returns
+        -------
+        str
+            Pathspec of the origin run or None
+        """
+        task = next(self.steps()).task
```

Comment:
Rare but maybe check if this returns ok because if there is no step at all (for example killed too early), I think this may fail.